library/core/src/char/methods.rs

```diff
@@ -9,6 +9,21 @@ use crate::unicode::{self, conversions};
 use super::*;
 
 impl char {
+    /// The lowest valid code point a `char` can have, `'\0'`.
```

Comment:
Can we mention that `char::MIN..=char::MAX` has a "hole" in it due to surrogate codepoints? I think that's explained in more detail on the `char` type itself, so it would just need to be one sentence or two. (And then that same note should be on `char::MAX` too, although perhaps not until `char::MIN` is stabilized.)